src/commands/checkModel.ts

```diff
@@ -44,6 +45,20 @@ export async function checkModel(
     if (!uri) {
         return;
     }
+
+    // Unless the given filePath already starts with 'MC', prepend MC to the name
+    // and check if it exists. If yes, it becomes the spec file.  If not, fall back
+    // to the original file.
+    const b = Utils.basename(uri);
+    if (!b.startsWith('MC')) {
+        const n = uri.toString().replace(b.toString(), 'MC' + b);
```

Comment:
This replacement looks very unreliable. What if, for example, the spec file is `/my_specs/spec.tla`? We'll end up searching for `/my_MCspecs/MCspec.tla`, which is incorrect.